arch/TriCore/TriCoreInstPrinter.c

```diff
@@ -44,9 +39,41 @@ static void printOperand(MCInst *MI, int OpNum, SStream *O);
 
 #include "TriCoreGenRegisterInfo.inc"
 
-static bool fill_mem(MCInst *MI, unsigned int reg, int32_t disp);
+static int32_t to_u32(int64_t x)
```

Comment:
It says `u32` but casts to signed `int32_t`. Shouldn't it be `i32` or something?